pkg/lang/python/plugin_persist.go

```diff
@@ -496,12 +494,10 @@ func (p *persister) queryFS(file *core.SourceFile, annotation *core.Annotation,
 		return nil
 	}
 
-	varName := ""
-	if fsSpecImport.Alias != "" {
-		varName = fsSpecImport.Alias
-	} else if len(fsSpecImport.ImportedAttributes) == 0 {
-		varName = fsSpecImport.Name
-	} else {
+	varNames := fsSpecImport.UsedAs
+	if len(varNames) == 0 {
+		// this means it's an attribute-import: "from aiofiles import f", which we don't support
+		log.Warn("Unsupported import")
```

Comment:
Good comment - could this go into the warning instead?

Reply:
I didn't do that because we output the annotation along with the log anyway, so it felt like it'd feel redundant. It would be something like:

```
Unsupported import: from aiofiles import f
| @klotho::persist {
|   id = 'hello'
| }
from aiofiles import f
```

What about something like: `Unsupported import. Please use "import aiofiles" without "from"`?

Reply:
☝️ I went with that